docs/install.md

```diff
@@ -122,6 +122,37 @@ Select the following tabs for the installation description of your target.
 
     is needed. A recommended size is at least a free space of 10 GiB.
 
+    ??? important "Installation of VirtualBox with VirtualBox extension pack ("Guest additions")"
+        If you plan to install the VirtualBox extension pack, some things have to be taken into account to achieve a working time synchronization. In general:
+
+        - The VirtualBox expansion pack contains an own time sync functionality, which can keep the VM system time in sync with the host system time.
+          The time sync mode in `dietpi-config` should hence be switched to **Custom** to avoid conflicts or timeouts during the boot process.
+        - VirtualBox without the extension pack requires a network time sync solution, e.g. one of the `dietpi-config` configurable time sync modes (which are based on `systemd-timesyncd`).
+
+        The needed additional installation steps (use of the extension pack) are described here (installation steps on the host system) and below (installation steps within the DietPi guest system).  
+        The installation on the host system contains the base VirtualBox installation and the installation of the extension pack. The VirtualBox host installation is described in the case of a Linux based host system (the description assumes a root user, otherwise add `sudo` appropriate).
+
+        1. Installation of VirtualBox on a Linux host system  
+           Following is the description of a manual installation (you need to change this depending on the actual VirtualBox release):
+
+             ```sh
+             mkdir ~/Downloads
+             cd ~/Downloads
+             wget https://download.virtualbox.org/virtualbox/6.1.18/virtualbox-6.1_6.1.18-142142~Debian~buster_amd64.deb
+             apt install ./virtualbox-6.1_6.1.18-142142~Debian~buster_amd64.deb
+             ```
+
+        2. Installation of the VirtualBox extension pack on a Linux host system  
+           Following is the description of a manual installation (you need to change this depending on the actual VirtualBox release):
+
+             ```sh
+             cd ~/Downloads
+             wget https://download.virtualbox.org/virtualbox/6.1.18/Oracle_VM_VirtualBox_Extension_Pack-6.1.18.vbox-extpack
+             VBoxManage extpack install Oracle_VM_VirtualBox_Extension_Pack-6.1.18.vbox-extpack
```

Comment:
Does the above extension pack install include making the `/dev/sr0` disc with guest additions available to the guest? At least on a Windows host it needs to be added from the GUI with the "Insert Guest Additions CD image" option from the VM window menu.

Reply:
@MichaIng 
I described this in the lower section (search for "Check whether an optical disc" to find this section).  
If the description is not easy enough I can easily add 1-2 screenshots to show it.